samples/web/content/apprtc/js/appwindow.js

```diff
@@ -61,10 +61,12 @@ var params = {
 // Generate random room id and connect.
 params.roomId = randomString(9);
 params.roomLink =  'https://apprtc.appspot.com/room/' + params.roomId;
-params.roomServer = 'https://apprtc.appspot.com';
+params.server = 'https://apprtc.appspot.com';
 
-var joinRoomLink = document.querySelector('#room-link');
-joinRoomLink.href = params.roomLink;
-joinRoomLink.text = params.roomLink;
+var joinRoomLink = document.querySelector('#room-link-href');
+if (joinRoomLink) {
```

Comment:
I had made this change before I excluded appwindow.js, so it isn't actually needed right now, I reverted it. I haven't discovered how to manipulate the DOM yet with our test set up. The ways suggested for phantomJS don't work.